components/PlantItem.js

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
-import { View, Text, Image} from 'react-native';
+import { View, Text, Image, TouchableHighlight} from 'react-native';
 
 import styles from '../styles/styles';
 
-export default function PlantItem (props) {
+export default function PlantItem ({ title, image, searchNavigation }) {
   return(
-    <View>
-      {/* <Image style={styles.meetAPlant} source={require(props.image)} /> */}
-      <Text>{props.title}</Text>
-    </View>
+    <TouchableHighlight
```

Comment:
Again, great error handling. Glad we were able to figure out that TouchableHighlight needed to wrap everything here.